src/test/javascript/spec/component/participation-submission/participation-submission.component.spec.ts

```diff
@@ -175,7 +175,7 @@ describe('ParticipationSubmissionComponent', () => {
         };
         comp.participation = participation;
         comp.exercise = exercise;
-        expect(comp.getCommitUrl(submission)).to.be.empty;
+        expect(comp.getCommitUrl(submission)).to.be.undefined;
```

Comment:
I think it makes more sense to move this test and test the createCommitUrl() instead of the getter since the getter ist atm just a wrapper for the createCommitUrl()